src/sqlancer/postgres/PostgresErrors.java

```
@@ -0,0 +1,16 @@
+package sqlancer.postgres;
+
+import java.util.regex.Pattern;
+
+import sqlancer.common.query.ExpectedErrors;
+
+public final class PostgresErrors {
+
+    private PostgresErrors() {
+    }
+
+    public static void addExpressionErrors(ExpectedErrors errors) {
+        errors.addRegex(Pattern.compile("ERROR: *"));
```

"ERROR: *" is too general and would hide unexpected errors.